Src/FluentAssertions/NumericAssertionsExtensions.cs

```diff
@@ -507,11 +480,7 @@ public static class NumericAssertionsExtensions
                 maxValue = ushort.MaxValue;
             }
 
-            Execute.Assertion
-                .ForCondition(!(minValue <= actualValue && actualValue <= maxValue))
-                .BecauseOf(because, becauseArgs)
-                .FailWith("Did not expect {context:value} to be within {0} from {1}{reason}, but found {2}.",
-                    delta, distantValue, actualValue);
+            NoBeCloseToFailIfValueInsideBounds(!(minValue <= actualValue && actualValue <= maxValue), distantValue, delta, actualValue, because, becauseArgs);
```

Comment:
🤔 I suppose you meant _not be_? 
🙃 See earlier remark

Reply:
Name simplified.